libs/plugins/stream-client/src/lib/core/models/graph-and-items/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './items-creater';
```

Comment:
typo in all these file names: "creater" should be "creator". We might want to change them once and for all since the files are new.